tests/lax_numpy_test.py

```diff
@@ -1477,6 +1477,38 @@ def testPadEmpty(self):
     np.testing.assert_equal(arr, jnp_res[2:-3, 3:-1])
     np.testing.assert_equal(np_res[2:-3, 3:-1], jnp_res[2:-3, 3:-1])
 
+  def testPadKwargs(self):
+    modes = {
+        'constant': {'constant_values': 0},
+        'edge': {},
+        'linear_ramp': {'end_values': 0},
+        'maximum': {'stat_length': None},
+        'mean': {'stat_length': None},
+        'median': {'stat_length': None},
+        'minimum': {'stat_length': None},
+        'reflect': {'reflect_type': 'even'},
+        'symmetric': {'reflect_type': 'even'},
+        'wrap': {},
+        'empty': {}
+    }
+    arr = [1, 2, 3]
+    pad_width = 1
+
+    for mode in modes.keys():
```

Comment:
Maybe add an invocation with an invalid `mode`?

Reply:
Added test for this.